setup.cfg

```diff
@@ -86,6 +86,7 @@ install_requires =
     paramiko >= 2.5.0, < 3
     pathlib2; python_version<"3.2"
     pexpect >= 4.6.0, < 5
+    pluggy >= 0.7.1, < 1.0
```

Comment:
The development status of the `pluggy` library is set to beta (https://pypi.org/project/pluggy/ - [4 - Beta]). Are we sure we want to run with this given development of this library may break everything?

Reply:
If pytest and tox are OK with it, I guess we can run with it too.

Reply:
That may be, but in the name of stability using a beta lib seems risky; maybe its just mislabeled? That said, if we're aware of the risks and the rate of change isn't breaking tox and pytest frequently then, :neutral_face: